src/interface.jl

```diff
@@ -5,8 +5,10 @@ import ForwardDiff: Dual
 #       for exchange and correlation ... if it does not help, remove it again
 abstract type Functional{Family,Kind} end
 
-"""Return the family of a functional. Results are `:lda`, `:gga`, `:mgga` and
-`:mggal` (Meta-GGA requiring Laplacian of ρ)"""
+"""Return the family of a functional. Results are `:lda`, `:hyb_lda`, `:gga`,
+`:hyb_gga`, `:mgga`, `:hyb_mgga` and 
```

Comment:
That makes sense to me.  I wasn't sure how necessary it would be to propagate the distinction.  I will remove for now and revert those changes in the DFTK PR.

So I'll keep the hybrid distinction in the raw mapping in Libxc.jl (in the `FAMILYMAP` dictionary) but convert `:hyb_*` -> `:*` in the constructor of `LibxcFunctional` in DFTK.  I'll close this PR since I guess I don't need to touch anything here.  Sound good @mfherbst ?

Reply:
Yes sounds good to me, thanks!